mava/networks.py

```diff
@@ -108,7 +110,54 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> distr
             jnp.finfo(jnp.float32).min,
         )
 
-        return distrax.Categorical(logits=masked_logits)
+        return tfd.TransformedDistribution(
+            distribution=tfd.Categorical(logits=masked_logits), bijector=tfb.Identity()
+        )
+
+
+class ContinuousActionHead(nn.Module):
+    """Continuous Action Head"""
+
+    action_dim: int
+    minimum: float = -1
+    maximum: float = 1
```

Comment:
I wonder if it would be cleaner to just pass the env in here and then get the minimum and maximum from there.

Reply:
Yeah seems cleaner and we no longer need to pass actions_dim as well 🤔